alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-conditions/decision-condition/decision-condition.component.ts

```diff
@@ -56,10 +56,10 @@ export class DecisionConditionComponent implements OnInit, OnChanges {
 
   form = new FormGroup({
     securityAmount: this.securityAmount,
-    singleDate: this.singleDate,
```

Comment:
We also need this field.

Reply:
It's added at the end of the group:
form = new FormGroup({
    securityAmount: this.securityAmount,
    administrativeFee: this.administrativeFee,
    description: this.description,
    componentsToCondition: this.componentsToCondition,
    singleDate: this.singleDate,
  });